packs/feat-effects/effect-plague-rat-curse.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
We’re not planning to add additional tracking effects to the PF2e system. The effect should apply a specific behavior (e.g. add a bonus) to the actor.

Reply:
Since the curse effect has to be re-rolled for each healing effect, it cannot be represented by RE (at least not to as far as I know). I have therefore removed the effect.